Clinical-Genomics/scout
# Fix #2072 - small group assessment sharing

This PR adds a functionality: shadow manual assessments on variants view from a small case group, intended as a simple workaround e.g. for relapses, additional remote family members, exome+genome or possibly genome + RNA - or indeed small cohorts with similar phenotype

**How to test**:
1. on local, load another test case with different internal id and perhaps for simplicity display id (643595)
1. add a group tag to this case using the case group card under the case pedigree and link to the mirror case (643594)
![Screenshot 2021-01-19 at 10 50 59](https://user-images.githubusercontent.com/758570/105018071-d30c3280-5a44-11eb-95e6-07ffc207fd1a.png)
1. make a few different manual evaluations on the original case and make sure these are now visible on the new case:
![Screenshot 2021-01-19 at 10 52 43](https://user-images.githubusercontent.com/758570/105018134-e3bca880-5a44-11eb-8b68-9fd9a88ad89b.png)
1. and vice versa
![Screenshot 2021-01-19 at 10 53 28](https://user-images.githubusercontent.com/758570/105018145-e7e8c600-5a44-11eb-9633-66892e3b90f6.png)

For now, this is manual to try out the potential, but it could also be extended to on load via config and via CLI. Good ways to do this may include Orderportalen, CG and/or via Balsamic and MIP. Basically anytime the case group relation is made clear and it is obvious the user wants them linked.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
